mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -961,10 +960,28 @@ message GetNetworkStatusResponse {
     // Whether we are behind.
     bool is_behind = 4;
 
-    // The latest block info reported by a consensus node
-    consensus_common.LastBlockInfoResponse last_block_info = 5;
+    // The latest block info data reported by a consensus node
+    LastBlockInfo last_block_info = 5;
 }
 
+// Data about the network state and last block processed by the consensus network
+message LastBlockInfo {
+    // Block index
```

Comment:
:wrench: It looks like `2` was intentionally dropped when this got brought in here. I'm thinking it should be marked reserved to prevent accidental adding back in, https://protobuf.dev/programming-guides/proto3/#reserved
```suggestion
    reserved 2;
    
    // Block index
```

Reply:
I think this would be a breaking change to mobilecoind api, so we shouldn't do it. There are python scripts that use last block info to try to figure out what the network block version is

Reply:
i decided to just add the deprecated field back as it was, there's no good reason to remove it in this change.